src/core/utils/compatibility.js

```diff
@@ -103,6 +104,21 @@ export const isFenix = (userAgentInfo: UserAgentInfoType): boolean => {
   return false;
 };
 
+export const isFenixCompatible = ({
+  _config = config,
+  addon,
+}: {
+  _config: typeof config,
+  addon: AddonType | null,
+}): boolean => {
+  // It is compatible if it is in the special collection, but rather than
+  // query a collection, we store a list of add-on guids in a config key.
+  if (addon && _config.get('fenixCompatibleGuids').includes(addon.guid)) {
```

Comment:
Couldn't you return `addon && _config.get('fenixCompatibleGuids').includes(addon.guid)` directly?

Reply:
Flow was angry when I did that, which seems odd, as I believe I've done similar before.

When I use the code `return addon && _config.get('fenixCompatibleGuids').includes(addon.guid);`, Flow says:

```
Cannot return addon && _config.get(...).includes(...) because null [1] is incompatible with boolean [2].
[incompatible-return]

 [1] 109│   addon,
     110│ }: {
     111│   _config: typeof config,
     112│   addon: AddonType | null,
 [2] 113│ }): boolean => {
     114│   // It is compatible if it is in the special collection, but rather than
     115│   // query a collection, we store a list of add-on guids in a config key.
     116│   return addon && _config.get('fenixCompatibleGuids').includes(addon.guid);
     117│ };
     118│
     119│ export type IsCompatibleWithUserAgentParams = {|
```

Playing around with it in the browser, it appears that `null && true` returns `null` and `null && false` returns `null`, so I guess Flow is correct here. I'm sure I've done similar things before, but I guess not.  Using `return Boolean(addon && _config.get('fenixCompatibleGuids').includes(addon.guid));` works though, so I've updated it. Thanks Flow!

Reply:
>  Thanks Flow!

I see what you're doing here 😄